docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -245,34 +164,18 @@ use filesystem inspection instead: `ALLUXIO_WORKER_DATA_SERVER_DOMAIN_SOCKET_ADD
 and `ALLUXIO_WORKER_DATA_SERVER_DOMAIN_SOCKET_AS_UUID=true`. Short-circuit writes are then enabled if
 the worker UUID is located on the client filesystem.
 
-## FUSE
+TODO: Should we just always recommend using filesystem inspection? Are there downsides?
```

Comment:
@madanadit would it make sense to only discuss the filesystem inspection approach?